apps/cloudinary/src/index.js

```diff
@@ -77,7 +77,7 @@ function renderDialog(sdk) {
     insertHandler: data => sdk.close(data)
   });
 
-  instance.show();
+  instance.show({folder: {path: config.startFolder}});
```

Comment:
Older installations will not have `config.startFolder` and as a result we'll pass `{ folder: { path: undefined } }` to Cloudinary instance. I'm not sure how Cloudinary widget will handle it: to be on the safe side I'd check if `config.startFolder` holds a valid value and only then pass it.

Reply:
Agreed. We should also potentially add logic to check if the path seems correct. i.e. does it start with a `/`?

Reply:
hi, thanks for feedback, good point about missing value when upgrading app, it currently works for me as is with not setting the value in configuration at all but still check might be more prudent. As the default is set to "" in the parameter description  maybe it won't be evr set to "undefined"  but empty string instead? need to check

As for validating / - I'd leave it to user or to cloudinary. It works as it is and I don't like second guessing here what proper value should be. In fact I just checked now our installation and I have "uatmedia/-/media/Images" with no starting /  there and it goes into that folder just fine. Cloudinary docs linked above do not describe proper format. Also I can imagine empty string could go to previously used folder so I'd leave it to cloudinary to handle it in best way and avoid validation here.

BTW I'd be grateful if you'd comment the #70 too as in current state it is still not very usable for us. We'd like to have start folder and max number of linked assets set per content type (e.g. some content needs only one asset, some more).

Reply:
please check the fix